src/HealthChecks.Prometheus.Metrics/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+
+namespace HealthChecks.Prometheus.Metrics.Extensions {
+    public static class PrometheusHealthCheckMiddleware {
+        public static IApplicationBuilder UseHealthChecksPrometheusExporter(this IApplicationBuilder applicationBuilder, string metricsEndpointName = "/healthmetrics") 
```

Comment:
- Probably endpoint parameter name is better than metricsEndpointName ( is not the name, is the endpoint path)
- PathString will be also good idea

Reply:
I agree, endpoint sounds better :)